File: integration-tests/hibernate-reactive-panache/src/main/java/io/quarkus/it/panache/reactive/NamedQueryWith2QueriesRepository.java
package io.quarkus.it.panache.reactive;

import jakarta.enterprise.context.ApplicationScoped;

import io.quarkus.hibernate.reactive.panache.PanacheRepository;

@ApplicationScoped
public class NamedQueryWith2QueriesRepository implements PanacheRepository<NamedQueryWith2QueriesEntity> {
}

File: integration-tests/hibernate-reactive-panache-kotlin/src/main/kotlin/io/quarkus/it/panache/reactive/kotlin/NamedQueryWith2QueriesRepository.kt
package io.quarkus.it.panache.reactive.kotlin

import io.quarkus.hibernate.reactive.panache.kotlin.PanacheRepository
import jakarta.enterprise.context.ApplicationScoped

@ApplicationScoped
class NamedQueryWith2QueriesRepository : PanacheRepository<NamedQueryWith2QueriesEntity>
